src/scripts/toc-utils.ts

```
@@ -0,0 +1,105 @@
+}
+
+/**
+ * Construit une hiérarchie de table des matières à partir des headings d'Astro
+ * @param headings - Array des headings extraits par Astro
+ * @param minDepth - Profondeur minimale à inclure (défaut: 2)
+ * @returns Hiérarchie de table des matières
+ */
+export function buildTocHierarchy(
+  headings: AstroHeading[], 
+  minDepth: number = 2
+): TocHeading[] {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)